pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -39,6 +46,21 @@ def _start(self):
 			self._async_wait()
 			return
 
+		decompression_binary = varexpand(shlex_split(decomp_cmd)[0], mydict=self.env)
```

Comment:
Actually the indexing needs to happen outside the varexpand and there I need to catch if its an empty one.